hack/generated/pkg/testcommon/kube_test_context_envtest.go

```diff
@@ -32,11 +32,11 @@ func createEnvtestContext(perTestContext PerTestContext) (*KubeBaseTestContext,
 	environment := envtest.Environment{
 		ErrorIfCRDPathMissing: true,
 		CRDDirectoryPaths: []string{
-			"../config/crd/bases",
+			"../../../v2/config/crd/bases",
```

Comment:
Presumably this will also change further in an upcoming PR when this stuff moves too?

Reply:
yes